### PR TITLE
Validations models specs

### PR DESCRIPTION
**Pull request by Estherstarr**
In this PR, I implemented the following:
- [ ] Add the following validations:
   - For the User model:
       -  Name must not be blank.
       -  PostsCounter must be an integer greater than or equal to zero.
   - For the Post model:
       - Title must not be blank.
       - Title must not exceed 250 characters.
   - CommentsCounter must be an integer greater than or equal to zero.
      - LikesCounter must be an integer greater than or equal to zero.
- [ ] Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true
```
Comment (afanimartin) [inline app/models/post.rb]: - Please limit the title to a maximum of 250 characters only.

File: spec/models/comment_spec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# posts
+# id,user_id,comments_counter
+# 1,4,0+1=1+1=2
+# 2,4,0+1=1
+
+# comments
+# id,author_id,post_id
+# 1,4,1
+# 2,4,2
+# 3,5,1
```
Comment (afanimartin) [inline spec/models/comment_spec.rb]: - Comments make our code look ugly. Please remove unnecessary comments in the codebase.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Estherstarr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉 🎉 🎉 
- Professional templates used for PR and the readme.
- Necessary tests added for the models and all tests pass. Good job!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/post.rb (hunk 1 above)
Comment (Whoistolu) [reply app/models/post.rb]: - Well resolved @Estherstarr  👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Estherstarr 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Highlight
✔️ Model Spec Passes
✔️ Models have validation

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**